root/account/preferences.tt

```diff
@@ -1,7 +1,5 @@
 [% USE dt = Class('DateTime') %]
-[% WRAPPER "layout.tt" title=l("Preferences") full_width=1 %]
-
-<h1>[% l('Preferences') %]</h1>
```

Comment:
The Preferences page already has headings, so this isn't needed for prettiness - and it's not really useful otherwise.